api/.eslintrc

```diff
@@ -1,4 +1,6 @@
 ---
   globals:
-    # Services:
+    # Sails services are globally-accessible by name.
```

Comment:
I copied it from [Sails docs](http://sailsjs.org/documentation/concepts/services/creating-a-service):
> Simply save a Javascript file containing a function or object into your api/services folder. The filename will be used as the **globally-accessible** variable name for the service. For example, an email service might look something like this: